PackageViewModel/SignPackageViewModel.cs

```diff
@@ -0,0 +1,363 @@
+﻿using NuGet.Common;
+using NuGet.Packaging.Signing;
+using NuGetPackageExplorer.Types;
+using System;
+using System.IO;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace PackageExplorerViewModel
+{
+    public class SignPackageViewModel : ViewModelBase, IDisposable
```

Comment:
The ViewModel should have a property for the Timestamp Server. That can be stored in the application settings for reuse.